docs/integrations/gerrit/locations.md

```diff
@@ -0,0 +1,37 @@
+---
+id: locations
+title: Gerrit Locations
+sidebar_label: Locations
+description: Integrating source code stored in Gerrit into the Backstage catalog
+---
+
+The Gerrit integration supports loading catalog entities from Gerrit hosted gits. Entities can
+be added to [static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+To use this integration, add configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  gerrit:
+    - host: gerrit.company.com
+      apiBaseUrl: gerrit.company.com/gerrit
```

Comment:
this should have https:// on it

Reply:
And by the way an apiBaseUrl would typically be the URL that points to the root of its REST API, not a website root (as it says below). If you have a real need for a web base URL, you could make it just `baseUrl` instead.

Reply:
Noted and will be fixed when the docs are ready to be merged.